pkg/ctl/create/iamserviceaccount.go

```diff
@@ -40,6 +40,8 @@ func createIAMServiceAccountCmd(cmd *cmdutils.Cmd) {
 
 		fs.BoolVar(&overrideExistingServiceAccounts, "override-existing-serviceaccounts", false, "create IAM roles for existing serviceaccounts and update the serviceaccount")
 
+		fs.StringToStringVarP(&cfg.Metadata.Tags, "tags", "", map[string]string{}, `A list of KV pairs used to tag the AWS resources (e.g. "Owner=John Doe,Team=Some Team")`)
```

Comment:
Modifying `ClusterConfig.Metadata.Tags` would have no effect for this command as it's for attaching tags to the cluster's CloudFormation stack (which happens during cluster creation), not the CloudFormation stack created for IRSAs.

Reply:
Thank you for these details. I have resubmitted the PR under #1756.